src/layers/models/layersManager.ts

```diff
@@ -49,13 +60,24 @@ export class LayersManager {
     this.logger.log('info', `Successfully added layer: ${layerRequest.name}`);
   }
 
-  // check if requested layer name is already exists in mapproxy config file (layer name must be unique)
-  public isLayerNameExists(jsonDocument: IMapProxyJsonDocument, layerName: string): boolean {
-    try {
-      const publishedLayers: string[] = Object.keys(jsonDocument.caches);
-      return publishedLayers.includes(layerName);
-    } catch (error) {
-      throw new Error(error);
+  public addLayerToBest(layerToBestRequest: ILayerToBestRequest): void {
```

Comment:
we said we will remove the term "best" from our code, no?

Reply:
fixed in ee20559